src/App.tsx

```diff
@@ -1,24 +1,114 @@
-/* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useState, useEffect, useMemo } from 'react';
 import { UserWarning } from './UserWarning';
+import { Todo } from './types/Todo';
+import { Select } from './types/Select';
+import { getTodos } from './api/todos';
+import { Main } from './components/Main';
+import { Footer } from './components/Footer';
+import { Notification } from './components/Notification';
+import { Errors } from './types/Errors';
+import { Header } from './components/Header';
 
-const USER_ID = 0;
+export const USER_ID = 9960;
 
 export const App: React.FC = () => {
+  const [todoList, setTodoList] = useState<Todo[] | null>(null);
+  const [selectedFilter, setSelectedFilter] = useState(Select.ALL);
+  const [typeError, setTypeError] = useState<Errors | null>(null);
+  const [notificationError, setNotificationError] = useState(false);
+  const [tempTodo, setTempTodo] = useState<Todo | null>(null);
+  const [loadersTodosId, setLoadersTodoId] = useState<number[] | null>(null);
+
+  const filteringList = useMemo(() => {
+    const filter = todoList && todoList.filter((todo) => {
+      switch (selectedFilter) {
+        case Select.ACTIVE:
+          return !todo.completed;
+        case Select.COMPLETED:
+          return todo.completed;
+        default:
+          return true;
+      }
+    });
+
+    return filter;
+  }, [selectedFilter, todoList]);
+
+  const countItemLeft = () => {
+    let count = 0;
+
+    todoList?.forEach(todo => {
+      if (!todo.completed) {
+        count += 1;
+      }
+    });
+
+    return count;
+  };
+
+  useEffect(() => {
+    getTodos(USER_ID)
+      .then(setTodoList)
+      .catch(() => {
+        setTypeError(Errors.UPDATE);
+        setNotificationError(true);
+      });
+  }, []);
+
   if (!USER_ID) {
     return <UserWarning />;
   }
 
   return (
-    <section className="section container">
-      <p className="title is-4">
-        Copy all you need from the prev task:
-        <br />
-        <a href="https://github.com/mate-academy/react_todo-app-loading-todos#react-todo-app-load-todos">React Todo App - Load Todos</a>
-      </p>
-
-      <p className="subtitle">Styles are already copied</p>
-    </section>
+    <div className="todoapp">
+      <h1 className="todoapp__title">todos</h1>
+
+      <div className="todoapp__content">
+        <Header
+          USER_ID={USER_ID}
+          counterItemLeft={countItemLeft}
+          setTypeError={setTypeError}
+          setNotificationError={setNotificationError}
+          todoList={todoList}
+          setTempTodo={setTempTodo}
+          setTodoList={setTodoList}
+        />
+
+        {todoList && (
+          <>
+            <Main
```

Comment:
```suggestion
            <TodoList
```
it's more descriptive